mk-files/release.mk

```diff
@@ -77,10 +77,10 @@ gorelease:
 	$(eval token := $(shell (grep github.com ~/.netrc -A 2 | grep password || grep github.com ~/.netrc -A 2 | grep login) | head -1 | awk -F' ' '{ print $$2 }'))
 	
 	$(aws-authenticate) && \
-	rm -r dist/ && \
+	if [ -d dist/ ]; rm -r dist/; fi && \
```

Comment:
`rm -rf dist` should do this without the need for `if`. The `f` flag suppresses the `no file or directory` error.

Reply:
```suggestion
	rm -rf dist && \
```